elm-street.cabal

```diff
@@ -18,6 +18,7 @@ category:            Language, Compiler, Elm
 build-type:          Simple
 extra-doc-files:     README.md
                      CHANGELOG.md
+extra-source-files:  test/golden/oneType.json
```

Comment:
are you sure this is everything that is needed? I think for golden test also a frontend directory might be required?

Reply:
ok probably not because front-end test is decoupled and not declared in cabal file so that one won't is invisible to stackage or nixpkgs infra.